mozilla-lockbox/lockbox-ios
# OAuth upgrade path dialog and handling

resolves #496 

#### expected behavior:
- When upgrading from an app that has the old login path, users will be sent to the welcome screen
- On the welcome screen, the login disclaimer dialog box will be shown.
- Hitting continue brings users to FxA signin

#### TESTING NOTES:

To test this branch, you need to have a clean install of pre-OAuth Lockbox. To make sure you have a clean install:

1. Disconnect your account inside the OAuth-signed-in app to make sure that the Keychain is empty when installing.
2. Delete the app, and then install an older version without OAuth (current production is #1664 so this will be the most accurate).
3. Sign in to this older version, confirm email etc so that you see logins syncing.
4. optionally lock or leave the app unlocked
5. Go to BB and install the most recent build of this branch
6. hopefully see the expected behavior?
